README.md

```diff
@@ -138,15 +138,28 @@ The current maintainers of the CloudNativePG project are:
 - Jonathan Gonzalez (EDB)
 - Marco Nenciarini (EDB)
 - Armando Ruocco (EDB)
-- Philippe Scorsolini (upbound)
+- Philippe Scorsolini (Upbound)
 
 They are listed in the [CODEOWNERS](CODEOWNERS) file.
 
-## Adopters
+### CloudNativePG at KubeCon
```

Comment:
Nice. I also think we need to add this to the website (PR coming soon for that repo)